app/controllers/util/ExcelUtils.scala

```diff
@@ -26,7 +26,7 @@ object ExcelUtils {
       }
     }
 
-    ws.range(0, 0, 0, 100).style().bold().set()
+    ws.range(0, 0, 0, rows.head.length).style().bold().set()
```

Comment:
Will this give us an off by one? Ie, will we get one extra column. Not totally sure from looking at the library docs, but worth sanity checking whether we want `.length` or `.length - 1` here.

Reply:
Well spotted